old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -276,34 +299,25 @@ the earth.
 respectively.  The origin for NAD 27 is Meades Ranch in Kansas.
 * *ED50* -- European Datum 1950
 
-For more information, read
+For more information, check out
 *  <a href="http://help.arcgis.com/en/arcgisdesktop/10.0/help/index.html#/Datums/003r00000008000000/" target="_blank">ESRI's ArcGIS discussion of Datums.</a>
 *  <a href="https://www.nceas.ucsb.edu/~frazier/RSpatialGuides/OverviewCoordinateReferenceSystems.pdf" target="_blank">page 2 in M. Fraiser's CRS Overview.</a>
 
 
-> NOTE: All coordinate reference systems have a vertical and horizontal datum
+Note that all coordinate reference systems have a vertical and horizontal datum
 which defines a "0, 0" reference point. There are two models used to define
-the datum: **ellipsoid** (or spheroid): a mathematically representation of the shape of
-the earth. Visit
-<a href="https://en.wikipedia.org/wiki/Earth_ellipsoid" target="_blank">Wikipedia's article on the earth ellipsoid </a>  for more information and **geoid**: a
-model of the earth's gravitatinal field which approximates the mean sea level
-across the entire earth.  It is from this that elevation is measured. Visit
+the datum (we will not cover these concepts in this tutorial): 
+* **ellipsoid** (or spheroid): a mathematical representation of the shape of
+the earth. Visit <a href="https://en.wikipedia.org/wiki/Earth_ellipsoid" target="_blank">Wikipedia's article on the earth ellipsoid </a>for more information. 
+* **geoid**: a model of the earth's gravitational field which approximates the mean 
+sea level across the entire earth.  Elevation is measured from this. Visit 
 <a href="https://en.wikipedia.org/wiki/Geoid" target="_blank">Wikipedia's geoid
-article </a>for more information. We will not cover these concepts in this tutorial.
+article </a>for more information.
 
 
 
 ### Coordinate Reference System Formats
 
 There are numerous formats that are used to document a **CRS**. In the next tutorial
 we will discuss three of the commonly encountered formats including: **Proj4**,
-**WKT** (Well Known Text) and **EPSG**.
-
-
-***
-
-## Additional Resources
```

Comment:
@donal-at-NEON: Last comment! I put these additional resources near the top in the usual "Additional Resources" section

Reply:
Great, thank you!